File: PMN-6.5/Plugins/Lpp.Dns.General.Metadata/Content/Models/Terms/Sex.ts
/// <reference path="../../../../../Lpp.Mvc.Composition/Lpp.Mvc.Boilerplate/jsBootstrap.d.ts" />
/// <reference path="../Terms.ts" />

module RequestCriteriaModels {
    export interface ISexTermData extends RequestCriteriaModels.ITermData {
        Sex: SexTypes;
    }

    export enum SexTypes {
        NotSpecified = -1,
        Male = 1,
        Female = 2,
        Both = 3,
        Aggregated = 4,
    }
}

File: Plugins/Lpp.Dns.General.CriteriaGroupCommon/Models/Terms/Sex.cs
using System.Runtime.Serialization;

namespace RequestCriteria.Models
{
    [DataContract]
    public class SexData : TermData
    {
        [DataMember]
        public SexTypes Sex { get; set; }
    }

    [DataContract]
    public enum SexTypes
    {
        [EnumMember]
        NotSpecified = -1,
        [EnumMember]
        Male = 1,
        [EnumMember]
        Female = 2,
        [EnumMember]
        Both = 3,
        [EnumMember]
        Aggregated = 4,
    }
}
